frontend/src/locales/scripts/en.json5

```diff
@@ -1165,9 +1165,9 @@
       },
       tag: {
         audio: {
-          zero: "No audio files with the selected tag",
-          count: "{count} audio file with the selected tag|{count} audio files with the selected tag",
-          countMore: "Over {count} audio files with the selected tag",
+          zero: "No audio track files with the selected tag",
+          count: "{count} audio track file with the selected tag|{count} audio track files with the selected tag",
+          countMore: "Over {count} audio track files with the selected tag",
```

Comment:
Same note as above^. The use of "track(s)" will eliminate the need for the word "file(s)".